DeAceroDLabs/marketplace-frontend
# Fix/dmp 532 Shopping cart footer

## What is done

[[DMP-532]  Shopping cart footer](https://deacero.atlassian.net/browse/DMP-532)

## How is done

- Added a min height to `ShoppingCart` component so that it will stay end of the view.

## How to test

1. Run the project via `npm start`
2. /carrito/productos.
3. Delete products until you only have 1 left. 
4. You should see the footer at the end of the view.

## Evidence

Before:
![image](https://user-images.githubusercontent.com/81311646/186274403-b4e6e65a-c607-4e8d-87f4-9912debf2ce8.png)

After:
![image](https://user-images.githubusercontent.com/81311646/186274432-8d2ded59-2d03-4b68-92ae-271fcc5e6b98.png)
